CAIPs/caip-78.md

```diff
@@ -0,0 +1,74 @@
+---
+caip: 78
+title: Asset Reference for the Hedera nft namespace
+author: Danno Ferrin (@shemnon)
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/169
+status: Draft
+type: Standard
+created: 2021-11-01
+updated: 2021-11-01
+requires: 19, 75
+---
+
+## Simple Summary
+
+This document is about the details of the Hedera Token Service 'nft' asset
+namespace and reference for CAIP-19.
+
+## Abstract
+
+In CAIP-19 a general asset identification scheme is defined. This is the
+implementation of CAIP-19 for `nft` in the
+[Hedera Token Service](https://docs.hedera.com/guides/docs/integrating-a-hedera-token-service-token#hedera-token-service)
+representing non-fungible tokens.
+
+## Motivation
+
+See CAIP-19.
+
+## Specification
+
+### Token Asset Namespace
+
+The asset namespace is called `nft` hosted in the Hedera Token Service (HTS). It
+references HTS non-fungible tokens in the `hedera` namespace (see CAIP-xx).
```

Comment:
replace xx with the number - should be 76 as far as I see

Reply:
updated.